djautotask/models.py

```diff
@@ -164,7 +164,8 @@ def __str__(self):
 
     def update_at(self, **kwargs):
         api_client = api.TicketsAPIClient(
-            impersonation_id=kwargs.get('impersonation_id')
+            impersonation_id=kwargs.get('impersonation_id', None),
```

Comment:
The default return value of get if a key doesn't exist is None, so you can remove all the redundant None statements added in this PR.

Reply:
Oh,.. It is due to my recent work mixed w/ poor memory: `self.cleaned_data.pop('force_auth_check', None)`

Thanks for catching this. 

All fixed and ready for review. Hopefully no mistake from my side again.